src/support/borsh.rs

```diff
@@ -41,9 +41,25 @@ impl<const BITS: usize, const LIMBS: usize> BorshDeserialize for Uint<BITS, LIMB
 impl<const BITS: usize, const LIMBS: usize> BorshSerialize for Uint<BITS, LIMBS> {
     #[inline]
     fn serialize<W: io::Write>(&self, writer: &mut W) -> io::Result<()> {
-        // TODO: non-allocating and remove the `alloc` feature requirement
```

Comment:
oh i believe the "borsh requires alloc" feature dep can be removed from `Cargo.toml` now